Dockerfile

```diff
@@ -38,45 +66,70 @@ ENV RAD_VERSION="1.0.0-rc.10"
 RUN curl -O -L https://files.radicle.xyz/releases/$RAD_VERSION/radicle-$OS_TARGET.tar.xz.sig
+RUN curl -O -L https://files.radicle.xyz/releases/radicle-httpd/latest/radicle-httpd-$HTTPD_OS_TARGET.tar.xz.sig
 
 # Install the Radicle checksum file
 RUN curl -O -L https://files.radicle.xyz/releases/$RAD_VERSION/radicle-$OS_TARGET.tar.xz.sha256
+RUN curl -O -L https://files.radicle.xyz/releases/radicle-httpd/latest/radicle-httpd-$HTTPD_OS_TARGET.tar.xz.sha256
 
-# Verify the Radicle signature
+# Verify the Radicle signatures
 RUN ssh-keygen -Y check-novalidate -n file -s radicle-$OS_TARGET.tar.xz.sig < radicle-$OS_TARGET.tar.xz
+RUN ssh-keygen -Y check-novalidate -n file -s radicle-httpd-$HTTPD_OS_TARGET.tar.xz.sig < radicle-httpd-$HTTPD_OS_TARGET.tar.xz
 
-# Verify the Radicle checksum
+# Verify the Radicle checksums
 RUN sha256sum -c radicle-$OS_TARGET.tar.xz.sha256
+RUN sha256sum -c radicle-httpd-$HTTPD_OS_TARGET.tar.xz.sha256
 
-# Extract the Radicle pacakge
+# Extract the Radicle pacakges
 RUN tar -xvJf radicle-$OS_TARGET.tar.xz --strip-components=1 -C ~/.radicle
+RUN mkdir ~/.radicle-httpd && tar -xvJf radicle-httpd-$HTTPD_OS_TARGET.tar.xz --strip-components=1 -C ~/.radicle-httpd
+RUN chmod +x /home/${USER}/.radicle-httpd/bin/radicle-httpd
 
 # Set the execution permission for the extracted files
-RUN chmod u+x /home/seed/.radicle/bin/rad
-RUN chmod u+x /home/seed/.radicle/bin/radicle-node
-RUN chmod u+x /home/seed/.radicle/bin/git-remote-rad
+RUN chmod u+x /home/${USER}/.radicle/bin/rad
+RUN chmod u+x /home/${USER}/.radicle/bin/radicle-node
+RUN chmod u+x /home/${USER}/.radicle/bin/git-remote-rad
 
 ENV RAD_PATH_EXPORT='export PATH="$PATH:/home/$USER/.radicle/bin"'
 ENV RAD_PATH="/home/${USER}/.radicle/bin:${PATH}"
-ENV RAD_HOME=/home/seed/.radicle/seed/
-ENV RAD_ALIAS=creature-pigeon
+ENV RAD_HOME=/home/${USER}/.radicle/${USER}/
+ENV RAD_ALIAS=${NODE_NAME}
 
 # Uncomment this and have your own Radicle passphrase if you need to run locally
 # ENV RAD_PASSPHRASE=yoursecret
 
-ENV KEYS_DIR=/home/seed/.radicle/seed/keys/
-ENV REPO_DIR=/home/seed/.radicle/seed/creature-pigeon
-ENV RADICLE_REPO_STORAGE=/home/seed/.radicle/seed/storage/
-ENV REPO_NAME=creature-pigeon
-
-# Peers DID
-ENV RADICLE_PEER_ONE=did:key:z6MkmesFj9djBn5dyH4vMEAjZeBjCb1BYhKgBMtC2EeeknHn
-ENV RADICLE_PEER_TWO=did:key:z6MkttLTb5NFW3hdMcgTT6f7FW6m7gkE6kosR2uEhg8dJmb1
-ENV RADICLE_PEER_THREE=did:key:z6MknzVNznWdLv1Tj19pLzDsXF5D6SHhjWK6WiMCtop6FK4K
-
+ENV KEYS_DIR=/home/${USER}/.radicle/${USER}/keys/
+ENV REPO_DIR=/home/${USER}/.radicle/${USER}/${NODE_NAME}
+ENV RADICLE_REPO_STORAGE=/home/${USER}/.radicle/${USER}/storage/
+
+# Copy peers DID
+COPY peers.list $REPO_DIR
+
+# Setting up the UI explorer
+RUN git clone https://github.com/radicle-dev/radicle-interface.git radicle-explorer
+
+WORKDIR /home/${USER}/radicle-explorer
+# switch to explorer version compatible with the version of radicle-httpd
+RUN git checkout b105d06fae415769bd20b65f0f4346d40537be78 \
```

Comment:
I'm not so sure about this long string. Is it to checking out the commit version of the radicle-httpd?

Reply:
yes because the latest explorer is incompatible with the latest released httpd API.
bad release mgt processes. so it forces to checkout the compatible commit. which happens to be that one the latest compatible one.

Reply:
they don't even create releases of the explorer. their artifact repository only list the node binary and httpd.

https://radicle.xyz/download

Reply:
oh got it!